### DISCUSSION TITLE
Help! Difference between roaster temp and artisan software

### DISCUSSION THREAD

**Discussion by Linstant-Gourmet** [artisan-roaster-scope/artisan#1172] [General]
Good morning,
I have a TOPER TKMSX 10 natural gas, and the artisan software shows me TG temperatures much higher than on my roaster.
Do you have any ideas to help me out?

**Comment by MAKOMO** [comment]
If the temperature difference is much higher it could be that the machine returns its readings in Fahrenheit instead of Celsius. You can change the temperature mode per channel in the MODBUS config (menu Config >> Port, 3rd tab "MODBUS") from C to F and test if this helps. 

If the temperature difference is minor the issue might be a [FAQ](https://artisan-scope.org/help/#FAQ-Setup-9)

**Reply by Linstant-Gourmet** [reply]
Thanks for your feedback. The temperature is ticked in Celsius.
